README.md

```diff
@@ -56,6 +56,7 @@ List of helpful tmux links for various tutorials, plugins, and configuration set
 - [tmuxp](https://github.com/tmux-python/tmuxp) :computer: tmux session manager and python library
 - [tmuxpair](https://github.com/goerz/tmuxpair) Command line script for setting up a temporary tmux session for pair programming
 - [vim-tmux-navigator](https://github.com/christoomey/vim-tmux-navigator) Vim and tmux integration
+- [tmuxake] https://github.com/nkh/tmuxake) A side-pane manager for tmux
```

Comment:
The link is missing a (

Would you also mind moving it up the list above tmuxifier to keep the list alphabetical?

Reply:
will do